scripts/install_third_party.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+source $(dirname $0)/setup.sh || exit 1
+
+# Checking if pip is installed. If you are having
+# trouble, please ensure that you have pip installed (see "Installing Oppia"
+# on the Oppia developers' wiki page).
+echo Checking if pip is installed on the local machine
+if ! type pip > /dev/null 2>&1 ; then
+    echo ""
+    echo "  Pip is required to install Oppia-ml dependencies, but pip wasn't"
+    echo "  found on your local machine."
+    echo ""
+    echo "  Please see \"Installing Oppia\" on the Oppia developers' wiki page:"
+
+    if [ "${OS}" == "Darwin" ] ; then
+      echo "    https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Mac-OS%29"
+    else
+      echo "    https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Linux%29"
+    fi
+
+    # If pip is not installed, quit.
+    exit 1
+fi
+
+echo Installing third party libraries
+
+# Install scikit-learn
+SKLEARN_VERSION=0.18.1
```

Comment:
@AllanYangZhou there is problem wih requirements.txt because whenever we install pip with `--target` it always re-downloads the library. That's why we first have to check that whether they are installed or not manually. Also this libraries are installed in `third_party/LIB_NAME-VERSION` and not all of them are installed in same folder. This is not possible with requirements.txt.

But I agree that we should have a single file (like Oppia has metadata.json) which contains all required 3rd party libs and version. So that intallation script and vm_config script can use it from that file and we don't have to modify bunch of files just for adding new library.

I think we can use `requirements.txt` for this purpose. We will not use it directly with `pip` but read it line by line and process libraries in both vm_config and installation script.

What do you think?